frontend/test/metabase/scenarios/dashboard/click-behavior.cy.spec.js

```diff
@@ -92,4 +99,81 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       },
     });
   });
+
+  it.skip("should not change the visualization type in a targetted question with mapped filter (metabase#16334)", () => {
+    // Question 2, that we're adding to the dashboard
+    const questionDetails = {
+      query: {
+        "source-table": REVIEWS_ID,
+      },
+    };
+
+    cy.createQuestion({
+      name: "16334",
+      query: {
+        "source-table": PRODUCTS_ID,
+        aggregation: [["count"]],
+        breakout: [["field", PRODUCTS.CATEGORY, null]],
+      },
+      display: "pie",
+    }).then(({ body: { id: question1Id } }) => {
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: { id, card_id, dashboard_id } }) => {
+          cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
+            cards: [
+              {
+                id,
+                card_id,
+                row: 0,
+                col: 0,
+                sizeX: 12,
+                sizeY: 10,
+                visualization_settings: getVisualizationSettings(question1Id),
+              },
+            ],
+          });
+
+          cy.visit(`/dashboard/${dashboard_id}`);
+
+          cy.intercept("POST", `/api/card/${card_id}/query`).as("cardQuery");
+        },
+      );
+    });
+
+    cy.wait("@cardQuery");
+    cy.get(".cellData")
+      .contains("5")
+      .first()
+      .click();
+
+    cy.findByText("Rating is equal to 5");
```

Comment:
Let's make sure that it's connected to the old question too.
```suggestion
    cy.findByText("Rating is equal to 5");
    cy.findByText("Started from 16334");
```

Reply:
The phrase is broken down into span and a link so `findByText` is not going to work here. I'll update the repro in a second.

Reply:
Changed it. btw, the test will now fail even before it reaches the assertion about the visualization type:
![image](https://user-images.githubusercontent.com/31325167/120822029-46c35d80-c556-11eb-9028-2f746b8c60b3.png)